### ISSUE TITLE
Fabric version ignores changed config file

### ISSUE THREAD

**Issue by PixieCatSupreme** [cScotPlay/BasicNetherOres#24]
I made changes in the config file setting all generation settings to true, but even in a newly created world none of the non vanilla ores seem to spawn. Currently only vanilla based nether ores spawn and ores added by Tech Reborn. This is happening in  the 1.16.3-5.3.0 version.

Config file:
```
{
  "diamondOre": {
    "diamondMinHeight": {
      "comment": "Set Diamond Min Spawn Height (Default: 90)",
      "value": 90
    },
    "diamondVeinSize": {
      "comment": "Set Max Diamond Vein Size (Default: 8)",
      "value": 8
    },
    "diamondPerChunk": {
      "comment": "Set Diamond Spawn Chance (Default: 3)",
      "value": 3
    },
    "diamondGeneration": {
      "comment": "Generate Diamond Ore",
      "value": true
    },
    "diamondMaxHeight": {
      "comment": "Set Diamond Max Spawn Height (Default: 125)",
      "value": 125
    }
  },
  "leadOre": {
    "leadGeneration": {
      "comment": "Generate Lead Ore",
      "value": true
    },
    "leadMinHeight": {
      "comment": "Set Lead Min Spawn Height (Default: 40)",
      "value": 40
    },
    "leadVeinSize": {
      "comment": "Set Max Lead Vein Size (Default: 8)",
      "value": 8
    },
    "leadMaxHeight": {
      "comment": "Set Lead Max Spawn Height (Default: 110)",
      "value": 110
    },
    "leadPerChunk": {
      "comment": "Set Lead Spawn Chance (Default: 8)",
      "value": 8
    }
  },
  "coalOre": {
    "coalGeneration": {
      "comment": "Generate Coal Ore",
      "value": true
    },
    "coalPerChunk": {
      "comment": "Set Coal Spawn Chance (Default: 20)",
      "value": 20
    },
    "coalVeinSize": {
      "comment": "Set Max Coal Vein Size (Default: 17)",
      "value": 17
    },
    "coalMaxHeight": {
      "comment": "Set Coal Max Spawn Height (Default: 115)",
      "value": 115
    },
    "coalMinHeight": {
      "comment": "Set Coal Min Spawn Height (Default: 25)",
      "value": 25
    }
  },
  "copperOre": {
    "copperPerChunk": {
      "comment": "Set Copper Spawn Chance (Default: 20)",
      "value": 20
    },
    "copperMaxHeight": {
      "comment": "Set Copper Max Spawn Height (Default: 75)",
      "value": 75
    },
    "copperMinHeight": {
      "comment": "Set Copper Min Spawn Height (Default: 35)",
      "value": 35
    },
    "copperVeinSize": {
      "comment": "Set Max Copper Vein Size (Default: 8)",
      "value": 8
    },
    "copperGeneration": {
      "comment": "Generate Copper Ore",
      "value": true
    }
  },
  "aluminumOre": {
    "aluminumMaxHeight": {
      "comment": "Set Aluminum Max Spawn Height (Default: 110)",
      "value": 110
    },
    "aluminumVeinSize": {
      "comment": "Set Max Aluminum Vein Size (Default: 8)",
      "value": 8
    },
    "aluminumGeneration": {
      "comment": "Generate Aluminum Ore",
      "value": true
    },
    "aluminumMinHeight": {
      "comment": "Set Aluminum Min Spawn Height (Default: 75)",
      "value": 75
    },
    "aluminumPerChunk": {
      "comment": "Set Aluminum Spawn Chance (Default: 10)",
      "value": 10
    }
  },
  "lapisOre": {
    "lapisGeneration": {
      "comment": "Generate Lapis Ore",
      "value": true
    },
    "lapisPerChunk": {
      "comment": "Set Lapis Spawn Chance (Default: 2)",
      "value": 2
    },
    "lapisMinHeight": {
      "comment": "Set Lapis Min Spawn Height (Default: 30)",
      "value": 30
    },
    "lapisVeinSize": {
      "comment": "Set Max Lapis Vein Size (Default: 7)",
      "value": 7
    },
    "lapisMaxHeight": {
      "comment": "Set Lapis Max Spawn Height (Default: 60)",
      "value": 60
    }
  },
  "tinOre": {
    "tinVeinSize": {
      "comment": "Set Max Tin Vein Size (Default: 8)",
      "value": 8
    },
    "tinPerChunk": {
      "comment": "Set Tin Spawn Chance (Default: 20)",
      "value": 20
    },
    "tinMinHeight": {
      "comment": "Set Tin Min Spawn Height (Default: 40)",
      "value": 40
    },
    "tinMaxHeight": {
      "comment": "Set Tin Max Spawn Height (Default: 110)",
      "value": 110
    },
    "tinGeneration": {
      "comment": "Generate Tin Ore",
      "value": true
    }
  },
  "osmiumOre": {
    "osmiumVeinSize": {
      "comment": "Set Max Osmium Vein Size (Default: 8)",
      "value": 8
    },
    "osmiumGeneration": {
      "comment": "Generate Osmium Ore",
      "value": true
    },
    "osmiumPerChunk": {
      "comment": "Set Osmium Spawn Chance (Default: 8)",
      "value": 8
    },
    "osmiumMaxHeight": {
      "comment": "Set Osmium Max Spawn Height (Default: 75)",
      "value": 75
    },
    "osmiumMinHeight": {
      "comment": "Set Osmium Min Spawn Height (Default: 30)",
      "value": 30
    }
  },
  "silverOre": {
    "silverVeinSize": {
      "comment": "Set Max Silver Vein Size (Default: 9)",
      "value": 9
    },
    "silverGeneration": {
      "comment": "Generate Silver Ore",
      "value": true
    },
    "silverPerChunk": {
      "comment": "Set Silver Spawn Chance (Default: 4)",
      "value": 4
    },
    "silverMaxHeight": {
      "comment": "Set Silver Max Spawn Height (Default: 60)",
      "value": 60
    },
    "silverMinHeight": {
      "comment": "Set Silver Min Spawn Height (Default: 30)",
      "value": 30
    }
  },
  "nickelOre": {
    "nickelPerChunk": {
      "comment": "Set Nickel Spawn Chance (Default: 8)",
      "value": 8
    },
    "nickelGeneration": {
      "comment": "Generate Nickel Ore",
      "value": true
    },
    "nickelMaxHeight": {
      "comment": "Set Nickel Max Spawn Height (Default: 50)",
      "value": 50
    },
    "nickelMinHeight": {
      "comment": "Set Nickel Min Spawn Height (Default: 30)",
      "value": 30
    },
    "nickelVeinSize": {
      "comment": "Set Max Nickel Vein Size (Default: 4)",
      "value": 4
    }
  },
  "emeraldOre": {
    "emeraldVeinSize": {
      "comment": "Set Max Emerald Vein Size (Default: 4)",
      "value": 4
    },
    "emeraldPerChunk": {
      "comment": "Set Emerald Spawn Chance (Default: 2)",
      "value": 2
    },
    "emeraldMaxHeight": {
      "comment": "Set Emerald Max Spawn Height (Default: 55)",
      "value": 55
    },
    "emeraldGeneration": {
      "comment": "Generate Emerald Ore",
      "value": true
    },
    "emeraldMinHeight": {
      "comment": "Set Emerald Min Spawn Height (Default: 25)",
      "value": 25
    }
  },
  "redstoneOre": {
    "redstoneGeneration": {
      "comment": "Generate Redstone Ore",
      "value": true
    },
    "redstoneVeinSize": {
      "comment": "Set Max Redstone Vein Size (Default: 8)",
      "value": 8
    },
    "redstoneMinHeight": {
      "comment": "Set Redstone Min Spawn Height (Default: 80)",
      "value": 80
    },
    "redstonePerChunk": {
      "comment": "Set Redstone Spawn Chance (Default: 8)",
      "value": 8
    },
    "redstoneMaxHeight": {
      "comment": "Set Redstone Max Spawn Height (Default: 115)",
      "value": 115
    }
  },
  "uraniumOre": {
    "uraniumPerChunk": {
      "comment": "Set Uranium Spawn Chance (Default: 6)",
      "value": 6
    },
    "uraniumVeinSize": {
      "comment": "Set Max Uranium Vein Size (Default: 4)",
      "value": 4
    },
    "uraniumMaxHeight": {
      "comment": "Set Uranium Max Spawn Height (Default: 125)",
      "value": 125
    },
    "uraniumMinHeight": {
      "comment": "Set Uranium Min Spawn Height (Default: 90)",
      "value": 90
    },
    "uraniumGeneration": {
      "comment": "Generate Uranium Ore",
      "value": true
    }
  },
  "ironOre": {
    "ironMaxHeight": {
      "comment": "Set Iron Max Spawn Height (Default: 110)",
      "value": 110
    },
    "ironVeinSize": {
      "comment": "Set Max Iron Vein Size (Default: 8)",
      "value": 8
    },
    "ironGeneration": {
      "comment": "Generate Iron Ore",
      "value": true
    },
    "ironPerChunk": {
      "comment": "Set Iron Spawn Chance (Default: 20)",
      "value": 20
    },
    "ironMinHeight": {
      "comment": "Set Iron Min Spawn Height (Default: 40)",
      "value": 40
    }
  },
  "oreProtection": {
    "piglinGuard": {
      "comment": "If set to true Piglins will protect Nether Ores (Default: true)",
      "value": true
    },
    "protectionRange": {
      "comment": "Set range that Piglins will aggro when mining ores(Default: 16)",
      "value": 16
    },
    "protectedBlocks": {
      "comment": "Add Blocks to be protected by Piglins (Example: minecraft:glowstone, bno:netheremerald_ore, ect. )",
      "value": [
        "bno:netheremerald_ore",
        "bno:netherdiamond_ore",
        "bno:netherlapis_ore",
        "bno:netherredstone_ore",
        "minecraft:nether_gold_ore",
        "bno:nethersilver_ore",
        "bno:netheriron_ore",
        "bno:netherlead_ore",
        "bno:nethernickel_ore",
        "bno:nethercoal_ore",
        "bno:nethercopper_ore",
        "bno:netheraluminum_ore",
        "bno:nethertin_ore",
        "bno:netherosmium_ore",
        "bno:netheruranium_ore",
        "minecraft:glowstone",
        "minecraft:nether_quartz_ore"
      ]
    },
    "silkEffect": {
      "comment": "Set to false if you want the Piglins to attack when using Silk Touch Tools (Default: true)",
      "value": true
    }
  }
}
```

Mod list:
```
Adorn-1.11.1+1.16.3.jar
appleskin-mc1.16-fabric-1.0.11.jar
astromine-1.11.4+fabric-1.16.3.jar
autoconfig1u-3.2.2.jar
BasicNetherOres-1.16.3-5.3.0-Fabric.jar
bettergraves-0.4.1.jar
bunch-o-trinkets-1.0.1.jar
Cardinal-Components-API-2.6.0.jar
DisableCustomWorldsAdvice-1.1.jar
enderitemc-enderitemod-1.2.2-1.16.2.jar
expandedstorage-5.6.30+1.16.2.jar
ezpas-1.2.3-1.16.2.jar
fabric-api-0.24.0+build.411-1.16.jar
fabric-language-kotlin-1.4.0+build.1.jar
fabric-language-scala-0.3.2.12-fat.jar
fabricmod_VoxelMap-1.10.11_for_1.16.3.jar
fairenchanting-1.2.0.jar
grind-enchantments-1.1.3+1.16.3.jar
Harvest-fabric-1.16.1-1.2.12-26.jar
immersive-portals-0.36-mc1.16.3-fabric.jar
improved-stations-2.0.3.jar
indrev-1.6.11-BETA.jar
industrialexpansion-1.0.0-fabric.jar
inmis-1.3.0-1.16.2.jar
InventorySorter-1.7.4-1.16.jar
itemscroller-fabric-1.16.3-0.15.0-dev.20200912.221805.jar
LargeFluidTank-16.0.6-fabric.jar
libblockattributes-all-0.8.2.jar
linkedstorage-1.3.9-1.16.jar
lithium-fabric-mc1.16.3-0.5.6.jar
malilib-fabric-1.16.3-0.10.0-dev.21+arne.1.jar
modmenu-1.14.6+build.31.jar
modupdater-1.1.11+1.16.2-rc2.jar
MoEnchantments-1.13.7.jar
mooblooms-1.3.2.jar
nbtcrafting-2.0.0+mc1.16.2-pre1.jar
netherite_horse_armor_v0.4.3_1.16.3.jar
RebornCore-1.16-4.4.2+build.73.jar
RedPower-1.1.5.jar
rer-2.1.2.jar
respawnablepets-1.16.3-1.jar
RingsOfAscension-1.16.3-1.0.7.jar
RoughlyEnoughItems-5.6.1.jar
rswires-4.9.1.jar
sandwichable-1.1.2__1.16.2.jar
simplepipes-all-0.4.1.jar
TechReborn-1.16-3.5.4+build.122.jar
torohealth-1.16.3-fabric-1.jar
trinkets-2.6.5.jar
vanilla-hammers-2.0.8-1.16.2.jar
wolves-with-armor-1.5.2-1.16.2.jar
woodenhoppers-1.0.0.jar
```

**Comment by cScotPlay** [comment]
The issue is fixed in 1.16.3-5.3.1 version.  Please let me know if you continue to have this issue.

**Comment by PixieCatSupreme** [comment]
It works now! Thank you.
